### ISSUE TITLE
logFC turned out to be "-lnf"

### ISSUE THREAD

**Issue by bettycatherine** [metaOmic/MetaDE#14]
Hi,
I used SEURAT to identify conserved markers and it uses meta-analysis methods from the MetaDE R package, so I suppose this is a question about MetaDE. **Some of the genes' logFC turned out to be “-Inf”.** I looked up and cound not find a explaination for this. Would some one please tell me what this means and how to deal with it. Should I exclude these genes from my analysis? 

![image](https://user-images.githubusercontent.com/22362094/106830172-81aca600-66c8-11eb-92b0-03ced554e467.png)
![image](https://user-images.githubusercontent.com/22362094/106830213-9426df80-66c8-11eb-9523-ecd3b8f6e4f2.png)

Thank you very much!

Xue

**Comment by matianzhou** [comment]
Is it continuous data (and did you log transform the data?) or count data
(are there concerns of excessive zeros)? How about the sample size? Those
are some possible reasons that give inf logFC.

Best,
Charles

Tianzhou (Charles) Ma, PhD
Assistant Professor
Department of Epidemiology and Biostatistics
University of Maryland School of Public Health
2234M SPH Building #255
4200 Valley Drive
College Park, MD 20742
Tel: 301-405-6421
Email: tma0929@umd.edu
Website: (Department) https://sph.umd.edu/department/epib/bio/91666
               (Personal) https://matianzhou.github.io/

**Comment by bettycatherine** [comment]
Thank you, Charles, for your quick reply.
My data should be count data, and I used smaller data, (subset this cluster), and there was no -Inf.
So you are suggesting to use log-trasnformed data ?
Thank you again!

best,
Xue

**Comment by matianzhou** [comment]
No, you can definitely use the count data, but you need to make sure you
choose the right method, e.g. DESeq2, edgeR or limmaVoom methods (where we
call their packages). There is a chance that their tool (e.g. DESeq2) did
some shrinkage on log2FC for count data and turn to some Inf values when
there is excessive zeros (in all samples or in samples of one group). In
your case, there is also a chance that there is outlier values in your full
data but in your subset.

Best,
Charles

Tianzhou (Charles) Ma, PhD
Assistant Professor
Department of Epidemiology and Biostatistics
University of Maryland School of Public Health
2234M SPH Building #255
4200 Valley Drive
College Park, MD 20742
Tel: 301-405-6421
Email: tma0929@umd.edu
Website: (Department) https://sph.umd.edu/department/epib/bio/91666
               (Personal) https://matianzhou.github.io/
